plugins/language_make.lua

```diff
@@ -11,8 +11,8 @@ syntax.add {
     { pattern = "$[@^<%%?+|*]",           type = "keyword2" },
     { pattern = "$%(.-%)",                type = "symbol"   },
     { pattern = "%f[%w_][%d%.]+%f[^%w_]", type = "number"   },
-    { pattern = "%..*:",                  type = "keyword2" },
-    { pattern = ".*:",                    type = "function" },
+    { pattern = "^%..*:%s",               type = "keyword2" },
+    { pattern = "^.*:%s",                 type = "function" },
```

Comment:
Having a space after `:` doesn't seem to be required by make, so I don't think we can just ignore those cases.

Maybe we could try matching assignments?